### PR TITLE
BCGOVSSIMO-15 Precessing Proof Request view

### PR DESCRIPTION
**Pull request by nikita-zakharov-dsr**
# Summary of Changes

add view for proof request processing
image

# Related Issues

https://jira.dsr-corporation.com/browse/BCGOVSSIMO-15

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [ ] Updated documentation as needed for changed code and new or modified features;
- [ ] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/screens/ProofRequestAccept.tsx [hunk 1]
```diff
@@ -45,6 +36,73 @@ const ProofRequestAccept: React.FC<ProofRequestAcceptProps> = ({ visible, proofI
...
       marginTop: 30,
     },
+    delayMessageText: {
+      textAlign: 'center',
+      marginTop: 20,
+    },
+  })
+
+  switch (status) {
+    case ProofState.RequestReceived:
```
Comment (Artemkaaas) [inline core/App/screens/ProofRequestAccept.tsx:46]: The view must be integrated integrated on the screen from the Verifer side (`ProofRequesting`) not the Prover's

Comment (nikita-zakharov-dsr) [reply core/App/screens/ProofRequestAccept.tsx:45]: done

File: core/App/screens/ProofRequestAccept.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { useProofById } from '@aries-framework/react-hooks'
 import { useNavigation } from '@react-navigation/core'
```
Comment (Artemkaaas) [inline core/App/screens/ProofRequestAccept.tsx:3]: Change in this file should be reverted

File: core/App/screens/ProofRequesting.tsx [hunk 3]
```diff
@@ -25,6 +27,65 @@ const windowDimensions = Dimensions.get('window')
...
+      <View style={{ flex: 1, justifyContent: 'space-between' }}>
+        <View style={styles.messageContainer}>
+          <Text style={[TextTheme.headingThree, styles.messageText]} testID={testIdWithKey('SendingProofRequest')}>
+            {t('ProofRequest.RequestProcessing')}
+          </Text>
+        </View>
+        <View style={[styles.loaderContainer]}>
+          <SendingProof />
+        </View>
+        <View>
```
Comment (Artemkaaas) [inline core/App/screens/ProofRequesting.tsx:73]: Is this empty view needed?

**Review by Artemkaaas — CHANGES_REQUESTED** [review]

**Review by Artemkaaas — APPROVED** [review]

**Pull request merged**